### ISSUE TITLE
[Bug]: [Accessibility] Clicking on the search button is not vocalised correctly

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#714]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-android/issues) for duplicate or closed feature requests

### Your test device

Samsung A53

### OS Version

Android 14

### App version

1.0

### Describe the issue

Clicking on the search button is not vocalized correctly : 
- Talkaback is activated 
- the user click on button search 
- Talkback vocalise : error editing 

### Expected Behavior

Talkback should vocalise a relevant and clear alternative for each action 
exp :editing search field

**Comment by paulinea** [comment]
Sorry @afrahB I don't understand the issue. This is when the user arrive on the search screen? Is it the vocalization of the search field? 
On my side when I click on the search button in the top app bar, I arrive on the search screen but the first thing vocalized is the back arrow, not the text field.

**Comment by paulinea** [comment]
Cannot reproduce
